src/test/java/org/folio/edge/rtac/MainVerticleTest.java

```diff
@@ -405,4 +414,142 @@ public void testResponseShouldBeEmptyWith200StatusWhenRtacResponseIsInvalid() {
     var actual = Holdings.fromXml(xml);
     assertEquals(new Holdings(), actual);
   }
+
```

Comment:
I don't really understand why these tests were separated out in the first place.  The only difference between them and the other tests in the mainverticleTest is that they are testing for the format of the response.  I'm a bit surprised that this worked at all in it's previous configuration-I would not have thought that inheritance would automatically perform the test setup and teardown required in the child class without an explicit invocation of some kind. In any case, the tests were all failing for me because, you guessed it, test setup and teardown was not happening in the child file.